dotnetv3/SES/Actions/SESActionExamples.cs

```diff
@@ -0,0 +1,281 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Amazon.SimpleEmail;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace SESActions;
+
+/// <summary>
+///     Use the AWS SDK for .NET with Amazon Simple Email Service (Amazon SES)
+///     to verify and manage email and domain identities. This example
+///     was created using the AWS SDK for .NET version 3.7 and .NET Core 6.0.
+/// </summary>
```

Comment:
(optional) This is a tiny nitpick; why indent the summary? To my eye it looks weird.

Reply:
It was the default style, but it's inconsistent with other examples, so I updated to use no indents here. Fixed.